packages/salesforcedx-vscode-apex/snippets/apex.json

```diff
@@ -0,0 +1,176 @@
+{
+    "Class": {
+        "prefix": "class",
+        "body": [
+            "class ${1:Name}",
+            "{",
+            "    $0",
+            "}"
+        ],
+        "description": "Class"
+    },
+    "DescribeFieldResult": {
+        "prefix": "dfr",
+        "body": [
+            "${1:SObjectType}.SObjectType.fields.${2:SObjectField}.getDescribe()"
+        ],
+        "description": "Get DescribeFieldResult"
+    },
+    "DescribeSObjectResult": {
+        "prefix": "dsr",
+        "body": [
+            "${1:SObjectType}.SObjectType.getDescribe()"
+        ],
+        "description": "Get DescribeSObjectResult"
+    },
+    "Assert Fail": {
+        "prefix": "fail",
+        "body": [
+            "System.assert(false, $0);"
+        ],
+        "description": "False assertion"
+    },
+    "do...while loop": {
+        "prefix": "do",
+        "body": [
+            "do",
+            "{",
+            "    $0",
+            "} while (${1:true});"
+        ],
+        "description": "do...while loop"
+    },
+    "while loop": {
+        "prefix": "while",
+        "body": [
+            "while (${1:true})",
+            "{",
+            "    $0",
+            "}"
+        ],
+        "description": "do...while loop"
+    },
+    "for": {
+        "prefix": "for",
+        "body": [
+            "for (${1:variable} : ${2:list_or_set}) {",
+            "    $0",
+            "}"
+        ],
+        "description": "for loop"
+    },
+    "for soql": {
+        "prefix": "fors",
+        "body": [
+            "for (${1:variable} : [${2:SOQL}]) {",
+            "    $0",
+            "}"
+        ],
+        "description": "for loop"
+    },
+    "Reverse for loop": {
+        "prefix": "forr",
+        "body": [
+            "for (Integer ${1:i} = ${2:length} - 1; ${1:i} >= 0 ; ${1:i}--)",
```

Comment:
This formatting is weird. Why is the `{` on the next line vs how we did it for the other `for` loops?